[2022-11-20T13:15:24.435+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Call_Center_ETL_DAG.create_staging_table manual__2022-11-20T13:15:12.651027+00:00 [queued]>
[2022-11-20T13:15:24.519+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Call_Center_ETL_DAG.create_staging_table manual__2022-11-20T13:15:12.651027+00:00 [queued]>
[2022-11-20T13:15:24.531+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T13:15:24.533+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-20T13:15:24.534+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T13:15:24.906+0000] {taskinstance.py:1383} INFO - Executing <Task(MySqlOperator): create_staging_table> on 2022-11-20 13:15:12.651027+00:00
[2022-11-20T13:15:24.964+0000] {standard_task_runner.py:55} INFO - Started process 890 to run task
[2022-11-20T13:15:25.007+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Call_Center_ETL_DAG', 'create_staging_table', 'manual__2022-11-20T13:15:12.651027+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/call_center_dag.py', '--cfg-path', '/tmp/tmpzmfsqvru']
[2022-11-20T13:15:25.023+0000] {standard_task_runner.py:83} INFO - Job 167: Subtask create_staging_table
[2022-11-20T13:15:25.750+0000] {task_command.py:376} INFO - Running <TaskInstance: Call_Center_ETL_DAG.create_staging_table manual__2022-11-20T13:15:12.651027+00:00 [running]> on host 9eba6bc17a61
[2022-11-20T13:15:27.708+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Abdelrhman Yassein
AIRFLOW_CTX_DAG_ID=Call_Center_ETL_DAG
AIRFLOW_CTX_TASK_ID=create_staging_table
AIRFLOW_CTX_EXECUTION_DATE=2022-11-20T13:15:12.651027+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-20T13:15:12.651027+00:00
[2022-11-20T13:15:27.714+0000] {mysql.py:84} INFO - Executing: 
        create table staging_call(
            id              varchar(255),
            customer_name   varchar(255), 
            sentiment       varchar(255),
            csat_score      varchar(255),
            call_timestamp  varchar(255),
            reason          varchar(255),
            city            varchar(255),
            state           varchar(255),
            channel         varchar(255),
            response_time   varchar(255),
            call_minutes    int,
            call_center     varchar(255)
        );

    
[2022-11-20T13:15:27.812+0000] {base.py:71} INFO - Using connection ID 'mysql_conn_id' for task execution.
[2022-11-20T13:15:27.982+0000] {sql.py:315} INFO - Running statement: 
        create table staging_call(
            id              varchar(255),
            customer_name   varchar(255), 
            sentiment       varchar(255),
            csat_score      varchar(255),
            call_timestamp  varchar(255),
            reason          varchar(255),
            city            varchar(255),
            state           varchar(255),
            channel         varchar(255),
            response_time   varchar(255),
            call_minutes    int,
            call_center     varchar(255)
        );

    , parameters: None
[2022-11-20T13:15:28.587+0000] {sql.py:324} INFO - Rows affected: 0
[2022-11-20T13:15:28.712+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=Call_Center_ETL_DAG, task_id=create_staging_table, execution_date=20221120T131512, start_date=20221120T131524, end_date=20221120T131528
[2022-11-20T13:15:28.876+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-20T13:15:29.130+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
